### PR TITLE
Refactored Angular code. Added separate service,controller,directives

### PR DESCRIPTION
**Pull request by ManikantaKandagatla**

### CODE REVIEW

File: src/main/webapp/resources/js/Controller/applicantController.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+angular.module('applicantApp')
+.controller('mainController',['$http','applicantUtilService','$scope',function($http,$scope,applicantUtilService,$location,$sce){
+
```
Comment (arajajyothibabu) [inline src/main/webapp/resources/js/Controller/applicantController.js:5]: I think parametyer `applicantUtilService` ==> `$applicantUtilService` in function parameter

File: src/main/webapp/resources/js/Controller/applicantController.js [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+	
+	$scope.skillset=[];
+	$scope.empobj = {};
+	$scope.Allemps = [];
+	$scope.newprojects = [];
+	$scope.neweditprojects =[];
+	$scope.employee={};
+	$scope.empProjs = [];
+	$scope.empProj = {};
+
```
Comment (arajajyothibabu) [inline src/main/webapp/resources/js/Controller/applicantController.js:27]: `scope.varaibles` don't have readability. Use `camelCase`

File: src/main/webapp/resources/js/Controller/applicantController.js [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+	$scope.ViewSelected = false;
+	$scope.param ='';
+	$scope.searchCriteria='search By Name';
+	
+	$scope.techSkills={
+		"Languages":["C","C++","JAVA","Python","C#","Perl"],
+		"SouceControlTools":["Git","Clear Case","SVN"],
+		"WebTechnologies" :["html","css","Jquery","JSP","Angular JS","node.js","backbonejs","Ruby on Rails"],
+		"OperatingSystems":["Windows","Linux","Solaris","Unix","IBM","Redhat"]
+	};
```
Comment (arajajyothibabu) [inline src/main/webapp/resources/js/Controller/applicantController.js:49]: Better maintain separate Source to read this data. Or else make an API call to fetch this data from server itself.

**Review by arajajyothibabu — COMMENTED** [review]
If you wrote this code single handed. Then It's fine to say that you became the senior developer :+1:

File: src/main/webapp/resources/js/Controller/applicantController.js (hunk 3 above)
Comment (ManikantaKandagatla) [reply src/main/webapp/resources/js/Controller/applicantController.js:49]: Yes need to refactor more for applicant/interviewer Client APIs.. yes those variables need to be fetched from db... Techskills,expertese on initial get request for applicant/interviewer views need to be implemented.. 
But final idea is to implement client routing for views unlike present .jsp files. will modify  them using angular routes reducing burden on server for unnecessary fetch of empty views.

File: src/main/webapp/resources/js/Controller/applicantController.js (hunk 2 above)
Comment (ManikantaKandagatla) [reply src/main/webapp/resources/js/Controller/applicantController.js:27]: Will modify them 👍

**Pull request merged**